R/util.R

```diff
@@ -237,3 +237,42 @@ time_ago <- function(time, missing = "unknown time ago") {
     paste(format(Sys.time() - time, digits = 2), "ago")
   }
 }
+
+duration_to_minutes <- function(period) {
```

Comment:
pass a name here, also work out why fail_msg did not fail. Perhaps this line is not covered?